### PR TITLE
Add Galician language

### PR DESCRIPTION
**Pull request by jseijas**
Added the first contribution for the Galician language. The Galician language is one of the three co-oficial languages inside Spain.

### CODE REVIEW

**Comment by fhemberger** [comment]
Hi, thanks for your translation!

@jseijas Can you please resolve the conflict?
And can anyone please check the translation and give a +1?

File: locale/ga/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  tagline-lts: Recomendado para a maioría
+  changelog: Cambios
+  api: Documentación do API
+  version-schedule-prompt: Ou revise á
+  version-schedule-prompt-link-text: Axenda de LTS
+---
+
+Node.js® é un entorno de execución para JavaScript construído co [motor de Javascript V8 de Chrome](https://developers.google.com/v8/).
+Node.js usa un modelo de operacións E/S sen bloqueo e orientado a eventos, o que o fai lixeiro e eficiente.
+O ecosistema de paquetes de Node.js, [npm](https://www.npmjs.com/), e o maior ecosistema do mundo de librerías
```
Comment (yagoff) [inline locale/ga/index.md]: O 'e' da última parte da frase leva til ao ser do verbo ser.

File: locale/ga/security.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+[security@nodejs.org](mailto:security@nodejs.org).
+Este será recibido por un subconxunto do equipo central que lida con cuestións de seguridade.
+
+O seu correo será revisado en 24 horas, e recibirá unha resposta máis detallada para o seu correo en 48 horas,
+indicando os seguinte pasos do proceso do seu informe.
+
+Tras a resposa inicial ao seu informe, o equipo de seguridade fará o posible para mantelo informado sobre o progreso
+realizado na dirección á solución e o anuncio completo, e pode solicitar información adicional ou orientación sobre o
+problema informado.
+Estas actualizacións serán enviadas polo menos cada cinco días, na práctica vais ser probable que sexa cada 24-48
```
Comment (yagoff) [inline locale/ga/security.md]: na práctica 'vai' (non 'vais') ser probable...

File: locale/ga/site.json [hunk 3]
```diff
@@ -0,0 +1,174 @@
+{
+    "title": "Node.js",
+    "author": "Fundación de Node.js",
+    "url": "https://nodejs.org/ga/",
+    "locale": "ga",
```
Comment (yagoff) [inline locale/ga/site.json]: Por qué 'ga' e non 'gl' que é o código ISO?

File: locale/ga/site.json [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    "scrollToTop": "Volta o principio",
```
Comment (yagoff) [inline locale/ga/site.json]: Volta 'ao', ou 'ó'.

File: locale/ga/site.json [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+    "reportNodeIssue": "Informar dun problema con Node.js",
+    "reportWebsiteIssue": "Informar du problema co sitio",
```
Comment (yagoff) [inline locale/ga/site.json]: 'dun'

File: locale/ga/site.json [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+        "events": {
+          "link": "get-involved/events",
+          "text": "Eventos"
+        },
+        "conduct": {
+            "link": "https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#code-of-conduct",
+            "text": "Comportamento"
+        }
+    },
+    "security"   : { "link": "security",        "text": "Security" },
```
Comment (yagoff) [inline locale/ga/site.json]: Estes textos por qué non están traducidos?

**Comment by jseijas** [comment]
Hi @yagoff Changes made. Can you review again?

**Review by yagoff — APPROVED** [review]
Great job!

**Comment by sotayamashita** [comment]
@fhemberger It can be merged! :bow:

**Pull request merged**